### PR TITLE
Case 22096: Added vhacd-util and Oven to all installers.

### PR DESCRIPTION
**Pull request by NissimHadar**
https://highfidelity.fogbugz.com/f/cases/22096/

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18656/HighFidelity-Beta-PR15356-781949d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18656/HighFidelity-Beta-Interface-PR15356-781949d.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18656/HighFidelity-Beta-PR15356-781949d.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18656/HighFidelity-Quest-Beta-PR15356-781949d.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18656/HighFidelity-Beta-PR15356-781949d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18656/HighFidelity-Beta-Interface-PR15356-781949d.exe

File: tools/vhacd-util/CMakeLists.txt [hunk 1]
```diff
@@ -12,7 +12,9 @@ target_link_libraries(${TARGET_NAME} ${VHACD_LIBRARIES})
 
 setup_memory_debugger()
 
-if (UNIX AND NOT APPLE)
+if (WIN32)
+  package_libraries_for_deployment()
```
Comment (SamGondelman) [inline tools/vhacd-util/CMakeLists.txt]: Per the ticket, can you also add this for oven, and make sure they both appear in both client only and client + sandbox builds?

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18662/HighFidelity-Beta-PR15356-13f0b97.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18662/HighFidelity-Quest-Beta-PR15356-13f0b97.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18662/HighFidelity-Beta-PR15356-13f0b97.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18662/HighFidelity-Beta-Interface-PR15356-13f0b97.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18662/HighFidelity-Beta-PR15356-13f0b97.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18662/HighFidelity-Beta-Interface-PR15356-13f0b97.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18663/HighFidelity-Beta-PR15356-3e292c4.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18663/HighFidelity-Beta-Interface-PR15356-3e292c4.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18663/HighFidelity-Beta-PR15356-3e292c4.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18663/HighFidelity-Beta-Interface-PR15356-3e292c4.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18663/HighFidelity-Beta-PR15356-3e292c4.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18663/HighFidelity-Quest-Beta-PR15356-3e292c4.apk)

**Pull request merged**